spec/services/nebraska_hours_calculator_spec.rb

```diff
@@ -56,17 +56,19 @@
           first_check_in = first_attendance_date.to_datetime + 8.hours
           first_check_out = first_check_in + 5.hours + 10.minutes
           second_check_in = second_attendance_date.to_datetime + 8.hours
+          second_check_out = second_check_in + 8.hours
```

Comment:
@JonErvin this addresses your point about the automatically created schedule, same in the next test